src/main/java/in/zapr/druid/druidry/postAggregator/CustomBucketsPostAggregator.java

```diff
@@ -0,0 +1,29 @@
+package in.zapr.druid.druidry.postAggregator;
+
+import java.util.Set;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NonNull;
+
+@Getter
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class CustomBucketsPostAggregator extends DruidPostAggregator {
+	
+	
+	 private static final String CUSTOMBUCKET_POST_AGGREGATOR_TYPE = "customBuckets";
+	 private String name;
+	 private String fieldName;
+	 private Set<Float> breaks;
```

Comment:
In druid's source code this field is **float[]**. 
Have you verified that Set<Float> will not cause any problems while mapping and query built using this builder is executing successfully ?

Verify similarly for **probabilities** field of quantiles.

Reply:
Here is what i can see using latest Histogram Code:

```
final Set<Float> probabilites = new HashSet<Float>();
		probabilites.add(0.50F); // 50%
		probabilites.add(0.90F); // 90%
		probabilites.add(0.95F); // 95%
		probabilites.add(0.98F); // 98%
		probabilites.add(0.99F); // 99%
		final DruidPostAggregator quantilesAgg = QuantilesPostAggregator.builder()
				 .name("PageLoadTimePercentiles")
				 .fieldName("pgload_agg")
				 .probabilities(probabilites)
				 .build();
```
```
"postAggregations": [{
		"type": "quantiles",
		"name": "pgload",
		"fieldName": "pgload_agg",
		"probabilities": [0.5, 0.9, 0.95, 0.98, 0.99]
	}]
```